packages/twenty-front/src/modules/app/components/SettingsRoutes.tsx

```diff
@@ -349,7 +346,11 @@ export const SettingsRoutes = ({
         <Route path={SettingsPath.Roles} element={<SettingsRoles />} />
         <Route path={SettingsPath.RoleDetail} element={<SettingsRoleEdit />} />
       </Route>
-      <Route path={SettingsPath.Developers} element={<SettingsDevelopers />} />
+      <Route path={SettingsPath.APIs} element={<SettingsApiKeys />} />
+      <Route
+        path={SettingsPath.PlaygroundRouter}
+        element={<PlaygroundRouter />}
```

Comment:
Let's make sure to lazy load everything in settings and in particular the playground stuff, we don't want to slow down the app for everyone on first load with that